WordPress/gutenberg
# Fix: List type style (list bullet) is different when nested inside another list

## What?
The PR is intended to solve the issue of Latest Posts displaying inconsistent list style 
Ref: https://github.com/WordPress/gutenberg/issues/63604

## Why ? 

WordPress does not set the list type style for the list block or on blocks that uses a list element.
These blocks uses the browser default styles, and uses disc, followed by circle and then square for the nested lists.
This means that when the block is nested inside another list, the list type style starts from the circle, not the disc, and this makes the design inconsistent.

## How?
This PR adds necessary SCSS style to the `wp-block-latest-posts__post-full-content` class to display the nested list items properly in the latest posts block 
<img width="365" alt="image" src="https://github.com/user-attachments/assets/258049db-1675-427f-87f7-c24eb542ef23">
The `.wp-block-latest-posts__post-full-content` class targets list items (li) and sets their list style type based on their nesting level: disc for the first level, circle for the second, and square for the third.

## Testing Instructions
Add a list block. Example:

```

<ul class="wp-block-list">
<li>One: disc
<ul class="wp-block-list">
<li>nested: circle
<ul class="wp-block-list">
<li>two levels: square
<ul class="wp-block-list">
<li>three levels: square</li>
</ul>
</li>
</ul>
</li>
</ul>
</li>
</ul>

```
Save.
Add a latest posts block.
Enable the post content option, and select "show full post".
Set the number of items to 1.
Confirm that the list types styles are not the same.

## Screenshots or screencast 

#### Created a new post with nested list items: 
<img width="1470" alt="image" src="https://github.com/user-attachments/assets/af10d5bb-c7fe-440f-878d-be8e9e96a290">

#### Added the post inside another post using the Latest Post block 
<img width="1469" alt="image" src="https://github.com/user-attachments/assets/da7d5000-1a64-45e4-9d7d-bade887ce9a9">


#### Post after saving
<img width="1464" alt="image" src="https://github.com/user-attachments/assets/129ae5f2-717d-452d-8a17-53db1c1e8e59">
